cmd/aries-js-worker/main.go

```diff
@@ -93,7 +94,7 @@ func main() {
 		ready <- struct{}{}
 	}
 
-	select {}
+	time.Sleep(math.MaxInt64)
```

Comment:
this doesn't look right to me.

read from a channel instead.

Reply:
try a pattern similar to the done channel (`chan struct{}`)
`<- done`

Reply:
the channel creates deadlock if you have suggestion sent to me. I can try it. I tried with the channel it did not work.

Reply:
@troyronda it was the first thing I did`try a pattern similar to the done channel`